bytes.xcodeproj/project.pbxproj

```diff
@@ -748,7 +700,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
+				SWIFT_VERSION = 4.0;
```

Comment:
Is this supposed to be 4.0 instead of 4.1?

Reply:
As far as I know only the major version matters here. This is how this is shown in Xcode itself:

![bildschirmfoto 2018-06-05 um 15 51 27](https://user-images.githubusercontent.com/113456/40980332-6a98220a-68d8-11e8-8659-9987925f6e40.png)